index.js

```diff
@@ -711,6 +715,7 @@ module.exports = function (log, indexesPath) {
     )
   }
 
+  // FIXME: why do we need to export this? It's simple and only used in 1 test
```

Comment:
This goes together with querySeq. I think those two methods were something that I added in the beginning thinking they would be a good idea and now I'm not sure sure. I checked and never really used them, so I'd actually be in favour of just removing them.

Reply:
Oh, deleting code always feels good. Will do

Reply:
It was there because I thought we might need it for about and contacts, but I'm not sure sure, we can always add it again if its just a simple PR that removes it.